src/TSMapEditor/Mutations/Classes/CloneObjectMutation.cs

```diff
@@ -12,10 +12,10 @@ public class CloneObjectMutation : Mutation
     {
         public CloneObjectMutation(IMutationTarget mutationTarget, IMovable movable, Point2D clonePosition) : base(mutationTarget)
         {
-            this.objectToClone = (AbstractObject)movable;
-            if (!objectToClone.IsTechno() && objectToClone.WhatAmI() != RTTIType.Terrain)
-                throw new NotSupportedException(nameof(CloneObjectMutation) + " only supports cloning Technos and TerrainObjects!");
+            if (!Helpers.IsCloningSupported(movable))
+                throw new NotSupportedException(nameof(CloneObjectMutation) + " only supports cloning Technos, TerrainObjects and Cell Tags!");
```

Comment:
Keeping naming consistent with `TerrainObjects`.

```suggestion
                throw new NotSupportedException(nameof(CloneObjectMutation) + " only supports cloning Technos, TerrainObjects and CellTags!");
```